be/src/olap/rowset/rowset_tree.h

```diff
@@ -76,7 +76,7 @@ class RowsetTree {
     //
     // The returned pointers are guaranteed to be valid at least until this
     // RowsetTree object is Reset().
```

Comment:
please update the comment related to rowset_ids.

Reply:
done